brevbaker-dsl/src/main/kotlin/no/nav/pensjon/brev/template/TemplateParagraph.kt

```diff
@@ -31,6 +31,14 @@ class ParagraphOnlyScope<Lang : LanguageSupport, LetterData : Any> : ParagraphSc
     fun includePhrase(phrase: ParagraphPhrase<out Lang>) {
         phrase.apply(this)
     }
+
```

Comment:
Det er feil pakke deklarasjon for denne filen.

Reply:
Det gjelder alle filene i samme pakke. Tror det bør tas i egen commit.